app/javascript/modules/listingDetailsAside/ListingDetailsApply.tsx

```diff
@@ -1,85 +1,183 @@
+      >
+        {t("label.applyOnline")}
+      </LinkButton>
+      {process.env.COVID_UPDATE && (
+        <div className={"mt-4"}>
+          <Heading priority={4} className={"text-base text-gray-800 font-sans"}>
+            {t("listings.apply.covidUpdate")}
+          </Heading>
+          <div className={"text-gray-700 text-base mt-2"}>
+            {t("listings.apply.covidUpdateInfo")}
+          </div>
+        </div>
+      )}
+      {acceptingPaperApps && (
+        <>
+          <OrDivider bgColor={"white"} />
+          {ordinalHeader(1, t("listings.apply.getAPaperApplication"))}
+          {t("listings.apply.paperApplicationsMustBeMailed")}
+          <Button
+            className={"w-full mt-4"}
+            transition={true}
+            icon={"arrowDown"}
+            iconPlacement={"right"}
+            onClick={() => setPaperApplicationsOpen(!paperApplicationsOpen)}
+          >
+            {t("label.downloadApplication", {
+              lang: "",
+            })}
+          </Button>
+          {paperApplicationsOpen && acceptingPaperApplications && (
+            <div className={"flex w-full items-center justify-center flex-col"}>
+              {paperApplicationURLs(isListingRental).map((app) => {
+                return (
+                  <span className={"pt-4"}>
+                    <LinkButton
+                      href={app.fileURL}
+                      unstyled
+                      className={"m-0 no-underline"}
+                      newTab={true}
+                    >
+                      {app.languageString}
+                    </LinkButton>
+                  </span>
+                )
+              })}
+            </div>
+          )}
+        </>
+      )}
+    </SidebarBlock>
+  )
+
+  const submitPaperApplicationBlocks = (
     <>
-      <p className={"mb-2"}>{t("listings.noAvailableUnits")}</p>
-      <p>{t("listings.enterLotteryForWaitlist")}</p>
+      <SidebarBlock className={"bg-blue-200"}>
+        {ordinalHeader(2, t("listings.apply.submitAPaperApplication"))}
+        {t("listings.apply.includeAnEnvelope")}
+      </SidebarBlock>
+      <SidebarBlock
+        className={"bg-blue-200"}
+        style={"sidebarSubHeader"}
+        title={t("listings.apply.sendByUsMail")}
+      >
+        <div className={"mb-2 text-gray-900 text-base"}>
+          <MultiLineAddress
+            address={{
+              placeName: listing.Application_Organization,
+              street: listing.Application_Street_Address,
+              city: listing.Application_City,
+              zipCode: listing.Application_Postal_Code,
+              state: listing.Application_State,
+            }}
+          />
+        </div>
+        {t("listings.apply.applicationsMustBeReceivedByDeadline")}
+      </SidebarBlock>
+      <SidebarBlock className={"bg-blue-200"}>
+        {`${t("listings.apply.doNotApplyOnlineAndPaper")} ${t(
+          "listings.apply.doNotDoMultipleApplications"
+        )}`}
```

Comment:
hmm, concatenating strings is not a good idea. i think we should consolidate these two into one string and have it re-translated.

Reply:
Added a new string!